recipes/snorkel/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "snorkel" %}
+{% set version = "v0.7.0" %}
```

Comment:
I think version should not have the `v` prefix.

Reply:
done